tachyon/zk/plonk/examples/BUILD.bazel

```diff
@@ -57,7 +57,6 @@ tachyon_cc_library(
         ":simple_circuit_test_data",
         ":simple_lookup_circuit",
         ":simple_lookup_circuit_test_data",
```

Comment:
Could you include `container_util.h` here to be explicit?

Reply:
But `circuit_test.h` doesn't use any thing from `container_util.h`, Rather, I found that `circuit_test` target depends on the unused dependencies.